### ISSUE TITLE
Need of a11y test for the components.

### ISSUE THREAD

**Issue by Dhruvi16** [cc-archive/vocabulary-legacy#738]
## Description

Storybook has an a11y addon that I think should be added as it eases the a11y tests for each component and would also disclose the a11y problems in a better manner.

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by mcastre** [comment]
Hi @Dhruvi16 Id like to work on this issue!

**Comment by zackkrida** [comment]
Hi @mcastre and @Dhruvi16 I think this is an excellent idea and I would love to see `@storybook/addon-a11y` used on the `packages/storybook` package initially, and we can implement on vue later.

**Comment by mcastre** [comment]
@zackkrida yea storybook's a11y addon is pretty sweet! we use it in our enterprise design system at work. Can i be assigned to this issue before I begin?

**Comment by mcastre** [comment]
@zackkrida @Dhruvi16 PR created. https://github.com/creativecommons/vocabulary/pull/740
